docs/setup/linux.md

```diff
@@ -11,6 +11,12 @@ MetaDescription: Get Visual Studio Code up and running on Linux.
 
 ## Installation
 
+### Ubuntu and [snap supported distros](https://snapcraft.io/docs/core/install) 
+
+Install the snap on [snap supported](https://snapcraft.io/docs/core/install) systems from the store.
+
+    snap install vscode --beta --classic
```

Comment:
The name `code` is already taken in the snap store. I'll see if I can claim it for Microsoft Visual Studio Code but for the time being I've updated the snap to support the alias `code`.